src/Commands/D9ify/ProcessCommand.php

```diff
@@ -0,0 +1,694 @@
+use Pantheon\Terminus\Models\Site;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @name d9ify
+ * Class ProcessCommand
+ *
+ * @package Pantheon\Terminus\Commands\D9ify
+ */
+class ProcessCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    use ConfigAwareTrait;
+
+    /**
+     * @var string
+     */
+    public static $HELP_TEXT = [
+        "*******************************************************************************",
+        "* THIS PROJECT IS IN ALPHA VERSION STATUS AND AT THIS POINT HAS VERY LITTLE   *",
+        "* ERROR CHECKING. PLEASE USE AT YOUR OWN RISK.                                *",
+        "* The guide to use this file is in /README.md                                 *",
+        "*******************************************************************************",
+    ];
+
+
+    /**
+     * @var Directory
+     */
+    protected Directory $sourceDirectory;
+    /**
+     * @var Directory
+     */
+    protected Directory $destinationDirectory;
+
+    /**
+     * Clone a pantheon site and spelunk the contents to create new D9 site.
+     *
+     * @authorize
+     *
+     * @command d9ify:process
+     *
+     * @aliases d9p
+     *
+     * @param string $sourceSite
+     *   Pantheon Site name/ID.
+     * @param ?string $destinationSite
+     *   Pantheon Destination Site (optional)
+     *
+     * @return void
+     * @throws \JsonException
+     *
+     * @usage terminus d9ify:process {sourceSiteName}
+     * @usage terminus d9ify:process {sourceSiteName} {destinationSiteName}
+     */
+    public function process($sourceSite, $destinationSite = null)
+    {
+        $this->output()->writeln(static::$HELP_TEXT);
+        try {
+            // Handle Source Site.
+            if ($sourceSite instanceof Site) {
+                $sourceSiteObject = $sourceSite;
+            }
+            if (is_string($sourceSite)) {
+                $sourceSiteObject = $this->getSite($sourceSite);
+            }
+            if (isset($sourceSiteObject) && $sourceSiteObject instanceof Site) {
+                $this->getContainer()->add("sourceDir", Directory::class)
+                    ->addArgument(['site' => $sourceSiteObject]);
+                $this->sourceDirectory = $this->getContainer()->get("sourceDir");
+            }
+
+            // Create destination Site if not exists or value is null.
+            if ($destinationSite == null) {
+                $destinationSite = $sourceSiteObject->getName() . "-" . date("Y");
+            }
+            if ($destinationSite instanceof Site) {
+                $destinationSiteObject = $destinationSite;
+            }
+            if (is_string($destinationSite)) {
+                $destinationSiteObject = $this->sites()->get($destinationSite);
```

Comment:
@TeslaDethray hey, right here $destinationSiteObject and $sourceSiteObject are the same value. When I $this->getSite("sitename") a second time, why is it giving me the first response?